.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
```

Comment:
I don't think most people want your IDE config - could you remove the `.idea` files?

Reply:
I believe I've done that now.